### ISSUE TITLE
Resources may use a font specified by the resource itself.

### ISSUE THREAD

**Issue by birchamp** [unfoldingWord/translationCore#6971]
### User Story

As a translator, I want complex script GL resources (Scripture, TN, TW, TA) to use the proper font so that the text will be displayed as intended by the GL team.

### Features / Specifications

- [ ] The GL resource (Scripture, TN, TW, TA) itself can specify which font and font settings (size, special properties)
- [ ] If the font is not specified it will fall back to Noto Sans
- [ ] Wherever the resources are displayed should use the specified font

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by mannycolon** [comment]
@birchamp Are there GL resources in DCS that specify the font in their manifest?

**Comment by mannycolon** [comment]
@birchamp if so, do you have any examples?

**Comment by birchamp** [comment]
No, fairly certain none of them do.   Can @mannycolon and @jag3773 choose a standard for specifying the font? Or propose something and ask Jessie to approve it.

**Comment by mannycolon** [comment]
@birchamp ,
@cckozie Had suggested adding font size and CSS font features settings.

Thus, I proposed the following:
```json
"font": {
  "type": "Padauk", // prefer font type
  "types": ["Padauk", "Other possible  font type"], // All possible font types
  "font-feature-settings":  "\"wtr\" 1", // CSS font feature settings
  "size": "24px", // Prefer font size if available 
}
```

**Comment by mannycolon** [comment]
@birchamp @jag3773 @cckozie ^^^

**Comment by birchamp** [comment]
@mannycolon I was hoping @jag3773 would weigh in here, but he's been in meetings all week.  This looks good to me. I like the css and font size settings. Good call @cckozie .

**Comment by birchamp** [comment]
It was decided that this issue is to be postponed and #7037 will handle it for now.
